contents/blog/best-mobile-app-ab-testing-tools.mdx

```diff
@@ -0,0 +1,317 @@
+- **Get started without talking to sales:** <span className="text-red text-lg">✖</span>
+- **Best for:** Engineering and product teams
+
+![LaunchDarkly mobile app A/B tests](../images/blog/best-mobile-app-ab-testing-tools/launch-darkly.png)
+
+### What is LaunchDarkly?
+
+[LaunchDarkly](/blog/posthog-vs-launchdarkly) is built for enterprises wanting to follow software development best practices. This means A/B testing changes, managing features, de-risking releases, and coordinating deploys.
+
+The people who find LaunchDarkly most useful are engineering managers, site reliability engineers, and product managers. 
+
+### Which platforms does LaunchDarkly support?
+
+- **iOS:** <span className="text-green text-lg">✔</span>
+- **Android:** <span className="text-green text-lg">✔</span>
+- **Flutter:** <span className="text-green text-lg">✔</span>
+- **React Native:** <span className="text-green text-lg">✔</span>
+- **Web:** <span className="text-green text-lg">✔</span>
+- **Server:** <span className="text-green text-lg">✔</span>
+
+### How much does LaunchDarkly cost?
+
+Experimentation is available on LaunchDarkly's Pro plan, which costs $20 per seat, per month, but experiments are priced separately to this. As per LaunchDarkly's docs:
+
+> "Experiments are billed monthly based on the number of unique user keys in each experiment. To estimate your Experimentation billing costs, multiply the number of experiments you run each month by the number of unique users in your experiment audience."
+
+## 8. Taplytics
+
+- **Free tier:** <span className="text-red text-lg">✖</span>
+- **Open source:** <span className="text-red text-lg">✖</span>
+- **Transparent pricing:** <span className="text-red text-lg">✖</span>
+- **Includes product analytics:** <span className="text-red text-lg">✖</span>
+- **Get started without talking to sales:** <span className="text-red text-lg">✖</span>
+- **Best for:** Product and marketing teams
+
+![Taplytics mobile app A/B tests](../images/blog/best-mobile-app-ab-testing-tools/taplytics.jpeg)
+
+### What is Taplytics?
+
+[Taplytics](https://taplytics.com/) is an A/B testing platform focusing on mobile. A particularly handy feature is their no-code experiment builder, which enables non-technical users to create and run A/B tests.
+
+### Which platforms does Taplytics support?
+
+- **iOS:** <span className="text-green text-lg">✔</span>
+- **Android:** <span className="text-green text-lg">✔</span>
+- **Flutter:** <span className="text-red text-lg">✖</span>
+- **React Native:** <span className="text-green text-lg">✔</span>
+- **Web:** <span className="text-green text-lg">✔</span>
+- **Server:** <span className="text-green text-lg">✔</span>
+
+### How much does Taplytics cost?
+
+Taplytics don't share their pricing publicly, so it's not clear how much it costs. You need to talk to sales first.
+
+## 9. Apptimize 
+
+- **Free tier:** <span className="text-red text-lg">✖</span>
```

Comment:
Seems like they do have a free tier: https://apptimize.com/plans/

Reply:
Only for feature flags, not for A/B tests